### ISSUE TITLE
r_list_length assertion failed

### ISSUE THREAD

**Issue by BlaCoiso** [radare/radare2#11881]
### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | Windows/x86/64
| File format of the file you reverse (mandatory)      | none
| Architecture/bits of the file (mandatory)            | none
| r2 -v full output, **not truncated** (mandatory)     | radare2 3.0.0 1 @ windows-x86-64 git. commit: 7d32c05f9a4134e78b5b7d5737ff9bcbcdfca54d build: Sat 10/20/2018__10:35:45.06

### Expected behavior
no warnings/errors
### Actual behavior
`WARNING: r_list_length: assertion 'list' failed (line 55)`
### Steps to reproduce the behavior 
run `r2 -c ia -`

**Comment by BlaCoiso** [comment]
looks like this might be a duplicate of #11844

**Comment by davidpolverari** [comment]
Even after #11844 is closed, this still happens. I will send a PR soon about it.
